### ISSUE TITLE
False positive for D417

### ISSUE THREAD

**Issue by PhilReinhold** [charliermarsh/ruff#792]
Running ruff v0.0.124, python 3.10.6, on OSX 12.6.1

Linting the following code:

```python
def f(x):
    """Do f.

    Args:
        x: the value

    Returns: the value
    """
    return x
```

Reports 
```
test.py:1:1: D417 Missing argument description in the docstring: `x`
```

The error goes away if the `Returns` block is reformatted with a new line.

```python
def f(x):
    """Do f.

    Args:
        x: the value

    Returns:
        the value
    """
    return x
```

**Comment by charliermarsh** [comment]
Great, thank you for the report!

**Comment by charliermarsh** [comment]
(For some reason we're not detecting NumPy-style docstrings in the first example, which seems to be the root cause.)

**Comment by charliermarsh** [comment]
(Err, wait, Flake8 doesn't detect NumPy-style docstrings either, so probably a bug in the Google-style parser.)

**Comment by PhilReinhold** [comment]
Thanks for the fix! However, I've noticed that the error returns if there is more than one argument present:

```python
# test.py
def f(x, y, z):
    """Do f.

    Args:
        x: the value
        y: Another thing
        z: A final argument

    Returns: the value
    """
    return x
```

```
$ ruff --version
ruff 0.0.177

$ ruff --select D test.py
Found 1 error(s).
test.py:1:1: D417 Missing argument descriptions in the docstring: `y`, `z`
```

The error again goes away if the `Returns` block gets a newline. Insisting on a newline there is actually fine for me, but the confusing error message is still unfortunate.
